Cookbook/Technical-Documents/XcodeTips.md

```diff
@@ -1,20 +1,70 @@
-## Tips and tricks
+# Xcode tips and tricks
 
-### Errors when pulling code/switching branches
+## Keybindings
+As we spend so much time in our ~~favourite~~ editor, it's essential to learn keybindings to decrease content switching and maintain focus.
+
+#### Navigating and finding things
+As long as the project is indexed, Xcode lets you jump to anything from anywhere.
+
+- **Cmd-Shift-0** to open the Apple documentation window.
```

Comment:
Not sure what would be the general consensus on this, but we could also use the actual symbols instead of plain text to describe the keys?
- ⌘ instead of **Cmd**
- ⌥ instead of **Opt**
- ⇧ instead of **Shift**
etc.

Reply:
We certainly could, but this was easier to type :) I can change it if there are more votes for this.

I personally find plain text more readable, and I don't like the reverse notation of **⌥⇧⌘K** that's used in the UI 🤷‍♂

Reply:
Fair point, I'm happy with this either way 👍